GliaWidgetsTests/Sources/AlertManager/AlertManager.Failing.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+@testable import GliaWidgets
+
+extension AlertManager {
+    static let failing = AlertManager(
+        environment: .failing,
+        viewFactory: .mock()
```

Comment:
`failing` entities should use only `failing` properties

Reply:
@EgorovEI Ideally yes. But that also depends if it is possible to create such failing entity without drastically rewriting it. For managed  dependencies (structs with closures) it is super easy, but for classes it's not always straightforward. 

@rasmustautsglia Since `AlertManager` is a class (reference type) it's not safe to have static let of this type, because it's internal state may affect other tests. I would recommend changing it to function instead, to always return new instance of the class. Also if viewFactory can't be easily adjusted to be failing, I would leave it as non-default parameter, like so:
```
static func failing(viewFactory: ViewFactory, environment: Environment = .failing) -> AlertManager {
...
```